modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -159,31 +159,31 @@ def wait(self):
         self.drain_call_queue()
         wait(self.future)
 
-    def mask(self, row_indices, col_indices):
+    def mask(self, row_labels, col_labels):
```

Comment:
nit: add type hints - presumably Optional[List[Hashable]] = None

Reply:
As per our discussion during today's (1/13/22) code-in, we should standardize {row|col}_indices -> {row|col}_labels, but we probably won't do it in this PR, since the scope is already quite large.